es6-promises/main.js

```diff
@@ -0,0 +1,10 @@
+import takeAChance from './take-a-chance.js';
+const chanceResult = takeAChance('Kevin');
+chanceResult.then(
+  (value) => {
+    console.log(value);
+  },
+  (reason) => {
+    console.error(reason.message);
+  }
+);
```

Comment:
This works, but it's best to use the `.then`, `.catch` chaining instead of using both a fulfill and reject handler in `.then`.